src/hotspot/cpu/arm/frame_arm.cpp

```diff
@@ -515,7 +515,7 @@ BasicType frame::interpreter_frame_result(oop* oop_result, jvalue* value_result)
       } else {
         obj = *(oop*)res_addr;
       }
-      assert(obj == NULL || Universe::heap()->is_in(obj), "sanity check");
+      assert(obj == NULL || Universe::is_in_heap(obj), "sanity check");
```

Comment:
Could have been changed to is_in_heap_or_null.

Reply:
Fixed